whitepaper/chapter01.md

```diff
@@ -0,0 +1,140 @@
+
+This fostered a “top-down” approach to telecommunications systems that were imperative rather than declarative. This approach was exceptionally resource intensive and expensive; redundancy literally meant acquiring multiple physical instances of required systems. Not surprisingly, this guiding architectural decision encouraged the tight bundling of numerous services and capabilities into individual physical systems. This resulted not only in inflated infrastructure and provisioning costs but also inefficient hyper-redundancy in system engineering for overlapping capabilities.
+
+<a name="1.5.2"></a>
+### 1.5.2 Software Defined Networking And The Emergence of VNFs
+
+In the late 1990s, the concept of virtualisation for enterprise computing garnered interest as enterprises tired of spending large sums of money on high-powered servers. Simultaneously, we saw the rapid rise in popularity of the commodity Open Source Linux Operating System. In 1999, VMware released its first virtualisation product for x86 devices. In the early 2010s, the concept of Network Functions Virtualisation (NFV) emerged as telcos and other users of big networking gear sought to virtualise their networking functionality, enhance resiliency, and take advantage of commodity hardware. The concept of NFV encapsulated Virtual Network Functions (VNFs) , Network Function Infrastructure (NFVI) and an orchestration component (MANO). The entire NFV hardware and software stack could be integrated and distributed across geographically dispersed data centres. In 2017, ONAP emerged as the open source standard platform for orchestrating and automating physical and virtual network elements with full lifecycle management. NFV and ONAP primarily focused on creating and orchestrating virtual analogs for physical equipment - virtual boxes.
+
+This was the same pattern followed during server virtualisation, when initial efforts focused on replicating physical servers as virtual entities. Cloud native however, requires a more fundamental redesign and rethink of how telco and backbone/core network systems function. In that sense, as well, cloud native for telco is actually not dissimilar from forklifting over monolithic legacy applications into virtual environments and then decoupling the tightly bundled functions and processes to create a more resilient, manageable, and performant infrastructure that derives immutability precisely from its ephemerality. In reality, CNFs will be introduced alongside VNFs and PNFs (physical network functions) and will need to fit into operational models that govern those earlier types of network functions. Additionally, CNFs will require the capability to be orchestrated by management systems already present in the service providers' networks.
```

Comment:
"Cloud native however, requires a more fundamental redesign and rethink of how telco and backbone/core network systems function."

> We should call out the reason for this. cloud native approaches changed how developers approach software design. As Network Functions become more software centric, they require the adoption of these paradigm shifts if they are to be cloud native themselves.

Reply:
"CNFs will require the capability to be orchestrated by management systems already present in the service providers' networks"

CNFs and cloud-native also provide the opportunity to decentralize complexity of LCM and service resiliency from the more "top-down" orchestration systems (and I include present ONAP in this). Just look at the difference in API frameworks from OpenConfig with gRPC versus the slew of interfaces in 3GPP and ETSI-NFV.

Reply:
What is still really important to understand that from a service perspective (ie an 5GSA core) Service Assurance still needs to be able to reference the CNF as an entity